checkov/cdk/checks/typescript/LambdaDLQConfigured.yaml

```diff
@@ -0,0 +1,48 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_116
+  name: Ensure that AWS Lambda function is configured for a Dead Letter Queue(DLQ)
+  category: GENERAL_SECURITY
+  framework: cdk
+scope:
+  languages:
+    - typescript
+definition:
+  or:
+    - pattern: new $PACKAGE.aws_lambda.Function(<ANY>, <ANY>, { <ANY> })
+      conditions:
+        - not_pattern: |
+            new $PACKAGE.aws_lambda.Function(<ANY>, <ANY>, { <ANY>, deadLetterQueue: <ANY>, <ANY>, deadLetterQueueEnabled: true ,<ANY>})
+        - metavariable: $PACKAGE
+          comparison: $PACKAGE == "aws-cdk-lib"
+    - pattern: new $PACKAGE.aws_lambda.CfnFunction(<ANY>, <ANY>, { <ANY> })
```

Comment:
Too many <ANY> (duplicated everywhere), fix it everywhere - the effect on the Engine performance is significant.

Reply:
this is not duplication this is different function name